20160826094924-sequelize-paper-trail-migration.js

```diff
@@ -14,7 +14,20 @@ var sequelizePaperTrailOptions = {
   enableCompression: false,
   enableMigration: false,
   enableStrictDiff: true,
-  continuationKey: 'userId'
+  continuationKey: 'userId',
+  // used when associating the revision model to user
+  belongsToUserOptions: {
+    foreignKey: 'userId',
+  },
+  // fields which can be provided in either in the opts.extraData
+  // OR
+  // CLS 'extraData' as object - keys will be looped over, and checked that they don't overwrite any existing ones in the revision model
+  // true/false shows whether the key is required or not
+  extraDataFields: {
+    projectId: false,
+    companyId: true,
+  },
+  extraDataContinuationKey: 'extraData',
```

Comment:
Let's leave this empty as well
```suggestion
  extraDataContinuationKey: null,
```